lobby-db/README.md

```diff
@@ -2,139 +2,70 @@
 
 A project that contains the migration scripts for the lobby postgres database.
 
-### Prod Deployments
-- Clone the project to lobby server
-- cd to the `lobby-db` project folder
-
-- run: `./prod_check_flyway.sh`
-  - Follow prompts and add user+password to creds file if not already done.
 
-First time you should see output like this:
-```
-triplea@triplea-server:~/lobby-db$ ./prod_run_flyway.sh 
- Error: Creds file does not exist at: /home/triplea/lobby.db.creds
-I have created the file for you, it contains:
-user=
-pass=
-
-Please update the file with valid database username and password and re-run this script
-```
 
-Now to run migrations:
-- run: `./prod_run_flyway.sh`
+### Prod Deployments
 
-If everything went well, it should all look like this:
-```
-triplea@triplea-lobby:~/lobby-db$ ./prod_run_flyway.sh 
-Using user: postgres
+We use [FlyWay](https://flywaydb.org/) to execute database changes. This means we do not change DB by hand,
+but instead check in the commands we wish to run and then deploy and run those commands
+with FlyWay.
 
-BUILD SUCCESSFUL in 1s
-1 actionable task: 1 executed
-triplea@triplea-lobby:~/work/triplea/lobby-db$ ./prod_check_flyway.sh 
 
-> Task :lobby-db:flywayInfo 
-Schema version: 1.06
-+-----------+---------+-------------------------------------+------+---------------------+---------+
-| Category  | Version | Description                         | Type | Installed On        | State   |
-+-----------+---------+-------------------------------------+------+---------------------+---------+
-| Versioned | 1.00    | create tables                       | SQL  | 2018-03-14 21:14:11 | Success |
-| Versioned | 1.01    | update tables                       | SQL  | 2018-03-14 21:14:11 | Success |
-| Versioned | 1.02    | add bcrypt password column          | SQL  | 2018-03-14 21:14:11 | Success |
-| Versioned | 1.03    | audit bans                          | SQL  | 2018-03-14 21:14:11 | Success |
-| Versioned | 1.04    | audit mutes                         | SQL  | 2018-03-14 21:14:11 | Success |
-| Versioned | 1.05    | add all user info to bans and mutes | SQL  | 2018-03-14 21:14:11 | Success |
-| Versioned | 1.06    | add access log                      | SQL  | 2018-03-14 21:14:11 | Success |
-+-----------+---------+-------------------------------------+------+---------------------+---------+
+- Production database changes are triggered by checking in
+  [flyway changes](https://github.com/triplea-game/triplea/tree/master/lobby-db/src/main/resources/db/migration)
+  followed by checking in an updated version number to infrastructure
+  [host_control.sh](https://github.com/triplea-game/infrastructure/blob/master/roles/host_control.sh)
 
+- Following that, when the [infastructure master branch] is merged to [infrastructure prod branch], then
```

Comment:
Were these intentionally meant to be links?  If so, the URI needs to be included.

Reply:
Good catch, looks like I never revisited those. They've been turned into links in: 0a1b4a2ca